### DISCUSSION TITLE
Volume managment

### DISCUSSION THREAD

**Discussion by JustCauseWhyNot** [wwmm/easyeffects#2648] [Q&A] [answered]
When using auto gain how do I wanna set the volume in places like applications, wireplumber, and most unknown to me is how do I wanna set the gain/volume knob on my motu m2? Do I wanna max everything out everywhere, or is there a more refined way of dealing with volumes?

**Comment by wwmm** [comment, marked as answer]
It is hard to give a procedure that will be good in every situation but I can tell what I do on my computers. I usually have the `autogain` and the `limiter` as the last plugins in my pipeline and I set the audio players built-in volumes to the maximum. After that I use only the system volume as volume control to fine tune the level I want. It has worked well for me.

**Reply by JustCauseWhyNot** [reply]
Do you have a amp, or audio interface of some kind where you adjust volume?

**Reply by wwmm** [reply]
>Do you have a amp, or audio interface of some kind where you adjust volume?

No. I use only the master volume control. The one GNOMe/KDE controls when you use its applet to change volume. In Pavucontrol it is shown as the volume control of the device being used as the system default.

My speakers have a volume control but I keep it always at the same level.

**Reply by JustCauseWhyNot** [reply]
Thank you for answering my questions.
